recipes/roocs-grids/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "roocs-grids" %}
+{% set package_name = "roocs_grids" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ package_name }}-{{ version }}.tar.gz
+  sha256: 30f48caa04308b88a265fcb331440af6304f10caea9b90801a5c8f9069e67070
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python >=3.8
+
+test:
```

Please also add this to ensure no new requirements will be missed in the future.
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check 
```